C-Plus-Plus/ds/Reversal algorithm for array rotation (GSSoC'21).cpp

```diff
@@ -0,0 +1,66 @@
+#include <bits/stdc++.h>
```

Comment:
rename the filename don't add gssoc or anything. Keep your file name as just the name of thing you have implemented and that too without spaces in the name.